flaskbb/forum/views.py

```diff
@@ -1108,6 +1110,115 @@ def post(self, mode=None):
         preview = renderer(text)
         return preview
 
+def user_get_file_max_length():
+    user_groups_file_max_length = current_user.primary_group.uploaded_file_max_length
+    take_default_value = True
+
+    if user_groups_file_max_length != 0:
+        take_default_value = False
+
+    if user_groups_file_max_length == -1:
+        return -1
+        
+    for group in current_user.secondary_groups:
```

Comment:
Можно сделать просто вот такой цикл, сразу со всеми группами:

```suggestion
    for group in [*current_user.secondary_groups, current_user.primary_group]:
```
Тогда код сильно упростится, основную группу не придется отдельно рассматривать